src/crimson/osd/main.cc

```diff
@@ -142,6 +167,9 @@ int main(int argc, char* argv[])
                                            hb_front_msgr.stop(),
                                            hb_back_msgr.stop());
         });
+        if (config.count("mkkey")) {
```

Comment:
crimson-osd should right away if `--mkkey | --mkfs`. but in this change, we always expect `--mkfs` or none.

Reply:
`TODO` comment?

Reply:
i will fix it in the next revision of this PR.